src/Data/String/CodePoints.purs

```diff
@@ -0,0 +1,216 @@
+module Data.String.CodePoints
```

Comment:
Could you add something along those lines as a doc-comment for the module please? Just so that people who come across it on Pursuit know what it's for and when it should be used.